src/meta/model_v2/src/source.rs

```diff
@@ -41,6 +41,7 @@ pub struct Model {
     pub version: i64,
     // `secret_ref` stores the mapping info mapping from property name to secret id and type.
     pub secret_ref: Option<SecretRef>,
+    pub rate_limit: Option<u32>,
```

Comment:
```suggestion
    pub rate_limit: Option<i32>,
```
`u32` is not allowed in model definition.